pkgs/tools/package-management/nix-prefetch-scripts/default.nix

```diff
@@ -10,6 +10,7 @@ let mkPrefetchScript = tool: src: deps:
 
     dontUnpack = true;
 
+    # TODO: nix-hash doesn’t need the whole nix closure bullshit
```

Comment:
I guess this was added by accident, right?

Reply:
yep, that wasn’t intended for public eyes :P